ddl/job_table.go

```diff
@@ -547,7 +560,7 @@ func AddBackfillJobs(sess *session, backfillJobs []*BackfillJob) error {
 		if err != nil {
 			return err
 		}
-		_, err = sess.execute(context.Background(), sql, label)
+		_, err = se.execute(context.Background(), sql, label)
```

Comment:
@zimulala I assume it should use the new session `se` from runInTxn instead of the outer `sess` here?

Reply:
Good catch.
This code resolves conflicts several times, which I should not have noticed. My mistake, thanks ~